compiler/src/Parse/String.hs

```diff
@@ -175,7 +175,7 @@ multiString pos end row _ _ sr sc =
     then Err sr sc E.StringEndless_Multi
     else
       let !word = P.unsafeIndex pos
-       in if word == 0x0A {- \n -}
+       in if word == 0x0A || word == 0x0D {- \n or \r -}
```

Comment:
This should probably be rewritten to check for the existance of `\r\n` and not just `\r`. You should probably also make a similar change to `multiStringBody` if you want this sequence to be valid within multistrings as well.